tests/Unit/ProtoTests/BasicExplicitPaginated/out/src/Gapic/BasicExplicitPaginatedGapicClient.php

```diff
@@ -0,0 +1,216 @@
+     *           {@see \Google\ApiCore\Transport\RestTransport::build()} methods for the
+     *           supported options.
+     *     @type callable $clientCertSource
+     *           A callable which returns the client cert as a string. This can be used to
+     *           provide a certificate and private key to the transport layer for mTLS.
+     * }
+     *
+     * @throws ValidationException
+     */
+    public function __construct(array $options = [])
+    {
+        $clientOptions = $this->buildClientOptions($options);
+        $this->setClientOptions($clientOptions);
+    }
+
+    /**
+     *
+     * Sample code:
+     * ```
+     * $basicExplicitPaginatedClient = new BasicExplicitPaginatedClient();
+     * try {
+     *     $aField = 'a_field';
+     *     $pageToken = 'page_token';
+     *     $partOfRequestA = [];
+     *     $response = $basicExplicitPaginatedClient->methodExplicitPaginated($aField, $pageToken, $partOfRequestA);
+     * } finally {
+     *     $basicExplicitPaginatedClient->close();
+     * }
+     * ```
+     *
+     * @param string           $aField
+     * @param string           $pageToken      Docs on this required standard page_token field will be ignored.
+     * @param PartOfRequestA[] $partOfRequestA Monolithic adds an import for a paginated method with a repeated message-typed field (!).
+     * @param array            $optionalArgs   {
+     *     Optional.
+     *
+     *     @type int $pageSize
+     *           Docs on this optional standard page_size field will be ignored.
+     *     @type PartOfRequestB[] $partOfRequestB
+     *     @type PartOfRequestC $partOfRequestC
+     *     @type RetrySettings|array $retrySettings
+     *           Retry settings to use for this call. Can be a {@see RetrySettings} object, or an
+     *           associative array of retry settings parameters. See the documentation on
+     *           {@see RetrySettings} for example usage.
+     * }
+     *
+     * @return \Testing\BasicExplicitPaginated\ExplicitResponse
+     *
+     * @throws ApiException if the remote call fails
+     */
+    public function methodExplicitPaginated($aField, $pageToken, $partOfRequestA, array $optionalArgs = [])
+    {
+        $request = new ExplicitRequest();
+        $request->setAField($aField);
+        $request->setPageToken($pageToken);
+        $request->setPartOfRequestA($partOfRequestA);
+        if (isset($optionalArgs['pageSize'])) {
+            $request->setPageSize($optionalArgs['pageSize']);
+        }
+
+        if (isset($optionalArgs['partOfRequestB'])) {
+            $request->setPartOfRequestB($optionalArgs['partOfRequestB']);
+        }
+
+        if (isset($optionalArgs['partOfRequestC'])) {
+            $request->setPartOfRequestC($optionalArgs['partOfRequestC']);
+        }
+
+        return $this->startCall('MethodExplicitPaginated', ExplicitResponse::class, $optionalArgs, $request)->wait();
```

Comment:
This one should also be returning a `PagedListResponse`, so something must be broken in `Utils\ExplicitPagination`